data/tools/wmllint

```diff
@@ -3055,10 +3055,6 @@ must be converted manually".format(filename, baseline + y + 1, old, name))
 
 # Generic machinery starts here
 
-def is_map(filename):
-    "Is this file a map?"
-    return filename.endswith(".map") or filename.endswith(".mask")
-
```

Comment:
I'm putting this on a line of a file so that it creates a Github thread, but it's surely not this line that's the cause.

Something has broken in wmllint. Without this PR, `data/tools/wmllint data/campaigns/Heir_To_The_Throne/` prints out about 500 lines of `... is not a known unit type`, seeing as I didn't include the core unit definitions. With this change, it prints out none of those lines.

With or without the change, it prints out `wmllint: spell check unavailable, install python-enchant to enable`, however that's expected.

Reply:
Working on it.

Reply:
Fixed. The cfg files were not being parsed because of a mistake here:
https://github.com/wesnoth/wesnoth/blob/bcddc90da4ac4c216b64024d20f27f5bc73dd467/data/tools/wesnoth/wmltools3.py#L17
I originally missed the comma at the end.